judge/chroot-settings.sh.in

```diff
@@ -0,0 +1,8 @@
+#/bin/sh
+
+# Chroot subdirs needed: (optional lib64 only needed for amd64 architecture)
+SUBDIRMOUNTS="etc usr lib bin opt"
```

Comment:
This is now duplicated from `chroot_startstop.sh`, so that should also import this.